styles.css

```diff
@@ -0,0 +1,106 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #3c3a39;
+  height: 80px;
+  width: 100%;
+  position: fixed;
+}
+
+.logo {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 23px;
+  line-height: 24px;
+  color: white;
+  margin: 35px;
+}
+
+.menu-icon {
+  margin: 20px;
+}
+
+.menu-line {
+  background-color: white;
+  width: 40px;
+  height: 5px;
+  margin: 6px;
+  border-radius: 3px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #1c1a19;
+  width: 100%;
+  height: 900px;
+}
+
+.headline-img {
+  position: absolute;
+  width: 500px;
+  left: 300px;
+  top: 450px;
+}
+
+.primary-text {
+  width: 327px;
+  height: 124px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  margin: 10px;
+}
+
+.supporting-text {
+  width: 327px;
+  height: 27px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
```

Comment:
- [ ] Kindly check out the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=kgjXcDYS4nMWzLhZ-0) and I would suggest we follow it strictly in styling our mobile version of the portfolio project. The `alignment`, `padding`, `margin`, and the use of `width` should be strictly taken cognizance of.
Kindly check out what we have in the template: 👇 

![corre](https://user-images.githubusercontent.com/78634841/204949012-7c0fe2de-fede-4ed2-9608-be4a7ad21a94.png)

But this is what i can currently see in your project: 👇 

![new](https://user-images.githubusercontent.com/78634841/204949090-c3f8cd5c-44ed-4e9c-9bf6-ad581b8f318e.png)

- [ ] It won't be cool of me if I didn't acknowledge the way you handle and style your `header navbar`. 🎉 🎉 🎉
 It would be great if you can use the same concept to style your `body section`